pkgs/development/python-modules/spacy/default.nix

```diff
@@ -31,6 +31,8 @@
 
 buildPythonPackage rec {
   pname = "spacy";
+  # when updating also run the update script for the models
+  # nix-shell maintainers/scripts/update.nix --argstr package python3Packages.spacy_models.en_core_web_sm
```

Comment:
Please add it to passthru.updateScript and remove this comment.

Reply:
If a package has no update script r-ryantm tries to use some default methods. 

I think you need to add an updater for spacy which would probably be nix-update.